packages/vscode-extension/src/builders/eas.ts

```diff
@@ -64,6 +64,13 @@ async function fetchBuild(config: EasConfig, platform: DevicePlatform) {
 
   const build = maxBy(builds, "completedAt")!;
 
+  if (!build.binaryUrl.endsWith(".apk") && !build.binaryUrl.endsWith(".apex")) {
+    Logger.error(
+      `EAS build artifact needs to be a development build in .apk or .apex format to work with the Radon IDE, make sure you set up eas to use "development" profile`
+    );
+    return undefined;
+  }
```

Comment:
I'm getting this error when trying to build my app on iOS, and from looking at the code, it seems that this check is run even on iOS EAS builds?

Reply:
thank you I added a platform check to avoid that in #796